src/test/java/io/anserini/search/GeoSearcherTest.java

```diff
@@ -0,0 +1,145 @@
+package io.anserini.search;
+
+import io.anserini.GeoIndexerTestBase;
+import org.apache.lucene.document.LatLonShape;
+import org.apache.lucene.document.ShapeField;
+import org.apache.lucene.geo.Line;
+import org.apache.lucene.index.DirectoryReader;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+import org.junit.Test;
+
+public class GeoSearcherTest extends GeoIndexerTestBase {
+
+    @Test
+    public void testGetLakeOntarioGeoJson() throws Exception {
+        Directory directory = FSDirectory.open(tempDir1);
+        DirectoryReader reader = DirectoryReader.open(directory);
+        IndexSearcher searcher = new IndexSearcher(reader);
+
+        Query q = LatLonShape.newBoxQuery("geometry", ShapeField.QueryRelation.INTERSECTS,
```

Comment:
We generally don't mind wide lines - I think this will look better on a single line. Same below?